l10n_br_hr_payroll/views/hr_payslip.xml

```diff
@@ -228,7 +228,7 @@
                     <field name="struct_id"
                            attrs="{'readonly': [('tipo_de_folha', '!=', 'rescisao')]}"
                            domain="[('tipo_estrutura', '=', tipo_de_folha)]" />
-                    <field name="data_pagamento_competencia"/>
+                    <field name="data_pagamento_competencia" attrs="{'required' : ['|', ('tipo_de_folha', '=', 'ferias'), ('tipo_de_folha', '=', 'rescisao')]}" />
```

Comment:
@lveras Quando são atributos de um mesmo campo, no caso um selection, você pode utilizar o 'in', i.e, ('tipo_de_folha', 'in', ['opcao_1', 'opcao_2']).

Reply:
vou substituir, nem me liguei, verdade.